docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -248,4 +248,18 @@ If the problem persists, then please contact Couchbase Technical Support.
 
 *Further Reading*: xref:7.0@server:metrics-reference:data-service-metrics.adoc#kv_ep_storage_age_second[Data Service Metrics], xref:7.0@server:install:sizing-general.adoc#sizing-data-service-nodes[Sizing Data Service Nodes]
 
+[#CB90071]
+=== Compaction Failure (CB90071)
+
+*Background*: Auto-Compaction can be set to run either at a specific time, or when bucket storage becomes too low.
+Auto-Compaction requires some free space in order to run, if there is none available then compaction cannot run.
+
+*Condition*: Auto-Compaction on bucket can fail if there is insufficient available storage.
+
+*Remediation*: Allocate more storage to the nodes in bucket, {{ $labels.bucket }} and attempt to rerun Auto-Compaction.
+You may also create a new cluster, with more storage, and use XDCR to transfer the files over to that, which should give you the space required for Auto-Compaction.
```

Comment:
Can remove "to that", and maybe specify unidirectional XDCR

Reply:
Fixed :)